### PR TITLE
added new query for Xray Tracing is not enabled for ansible

### LINKED ISSUE #1255: Add API Gateway Tracing query for Ansible
### Platform
Ansible

### Provider
Ansible

### Description
Xray Tracing is not enabled

### PR DESCRIPTION
**Pull request by soniantao**
Closes #1255

### CODE REVIEW

File: assets/queries/ansible/aws/api_gateway_tracing/query.rego [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  awsApiGateway := task["community.aws.aws_api_gateway"]
+  awsApiGateway.tracing_enabled == false
```
Comment (joaoReigota1) [inline assets/queries/ansible/aws/api_gateway_tracing/query.rego:8]: should also check for "no", "false"
 isAnsibleFalse(answer) {
       lower(answer) == "no"
 } else {
       lower(answer) == "false"
 } else {
       answer == false
 }

File: assets/queries/ansible/aws/api_gateway_tracing/query.rego [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  clusterName := task.name
+
+    result := {
+                "documentId":       input.document[i].id,
+                "searchKey":        sprintf("name={{%s}}.{{community.aws.aws_api_gateway}}.tracing_enabled", [clusterName]),
+                "issueType":        "IncorrectValue",
+                "keyExpectedValue": "community.aws.aws_api_gateway.tracing_enabled should be enable",
+                "keyActualValue":   "community.aws.aws_api_gateway.tracing_enabled is disable"
```
Comment (joaoReigota1) [inline assets/queries/ansible/aws/api_gateway_tracing/query.rego:16]: typo in "is disable" change to "is disabled"

File: assets/queries/ansible/aws/api_gateway_tracing/query.rego [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  task := tasks[t]
+  awsApiGateway := task["community.aws.aws_api_gateway"]
+  awsApiGateway.tracing_enabled == false
+  clusterName := task.name
+
+    result := {
+                "documentId":       input.document[i].id,
+                "searchKey":        sprintf("name={{%s}}.{{community.aws.aws_api_gateway}}.tracing_enabled", [clusterName]),
+                "issueType":        "IncorrectValue",
+                "keyExpectedValue": "community.aws.aws_api_gateway.tracing_enabled should be enable",
```
Comment (joaoReigota1) [inline assets/queries/ansible/aws/api_gateway_tracing/query.rego:15]: typo in "should be enable" change to "should be enabled"

**Review by joaoReigota1 — CHANGES_REQUESTED** [review]

File: assets/queries/ansible/aws/api_gateway_tracing/query.rego (hunk 1 above)
Comment (soniantao) [reply assets/queries/ansible/aws/api_gateway_tracing/query.rego]: you are correct. I missed it. I've already changed

**Comment by ruigomescx** [comment]
Closed due to identical query merged before (#1652)

**Pull request closed without merging**